pmd-java/src/main/resources/category/java/performance.xml

```diff
@@ -369,15 +369,29 @@ buf.append("1m");           // good
 String.trim().length() is an inefficient way to check if a String is really empty, as it
 creates a new String object just to check its size. Consider creating a static function that
 loops through a string, checking Character.isWhitespace() on each character and returning
-false if a non-whitespace character is found. You can refer to Apache's StringUtils#isBlank (in commons-lang),
-Spring's StringUtils#hasText (in the Spring framework) or Google's CharMatcher#whitespace (in Guava) for
-existing implementations.
+false if a non-whitespace character is found. A Smarter code to check for an empty string
+would be:
+<![CDATA[
```

Comment:
the whole description should be wrapped in `<![CDATA[` rather than this part alone